native/ios/Gemfile.lock

```diff
@@ -272,13 +273,14 @@ GEM
       xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
-  ruby
```

Comment:
Any idea why this only updates for ios but not the android or native? Same for `BUNDLED WITH` below...

Reply:
The platform should be readded!!
As i found out the bundler check the version of the current `Gemfile.lock` and uses this. When updating rubygems you have to remove the `Gemfile.lock` and rereun `bundle install` to get correct version and platforms.
Please do that for `native` & `android` & `ios`